lib/manageiq/api/common/users.rb

```diff
@@ -0,0 +1,70 @@
+module ManageIQ
+  module API
+    module Common
+      module Users
+        IDENTITY_KEY = 'x-rh-identity'.freeze
+        VALID_KEYS = %w(account_number user username email is_org_admin)
+
+        def self.current
```

Comment:
@syncrou This should return back a user object after it has parsed out the identity key from the header.
Preferably stored in a thread local variable